content/applications/finance/fiscal_localizations/peru.rst

```diff
@@ -942,3 +942,126 @@ a scheduled action, but you can also send each invoice manually if needed.
+inventory.
+
+Context
+~~~~~~~
+
+PLE 12.1: Permanent Inventory Record in Physical Units
+******************************************************
+
+This record **only tracks inventory in physical units**, focusing on the inflow and outflow of goods
+for effective management and planning. It must be maintained semi-annually (January-June and
+July-December) with monthly transaction details reported within these periods. The submission
+deadlines are October 1st for the first semester and April 1st for the second semester,
+in accordance with Superintendent Resolution 169-2015.
+
+PLE 13.1: Permanent Valued Inventory Record
+*******************************************
+
+This record tracks **both physical quantities and monetary values of inventory**, providing a
+comprehensive view for tax and management purposes. It must be maintained semi-annually
+(January-June and July-December) with monthly transaction details reported within these periods. The
+submission deadlines are October 1st for the first semester and April 1st for the second semester,
+in accordance with Superintendent Resolution 169-2015.
+
+Configuration
+~~~~~~~~~~~~~
+
+Before generating the PLE 12.1 or PLE 13.1 reports, it is recommended to
+:ref:`install and update <general/install>` all Peruvian modules (`l10n_pe`).
+
+Most importantly, install the **Peru - Stock Reports** module (`l10n_pe_reports_stock`).
+
+Update relevant fields
+**********************
+
+Once the **Peru - Stock Reports** module is installed, consider updating and completing the fields
+for:
+
+- :ref:`Products <peru/reports/products>`
+- :ref:`Warehouses <peru/reports/warehouses>`
+- :ref:`Inventory transfers <peru/reports/transfers>`
+
+.. _peru/reports/products:
+
+Products
+^^^^^^^^
+
+There are a few ways to configure a product for PLE reporting:
+
+- **Type of existence:** Navigate to: `Sales --> Product --> Product`. Select a product. In the
+  :guilabel:`Accounting` tab, select the :guilabel:`Type of Existence` according to SUNAT's Table 5
+  for inventory reporting.
+
+   .. image:: peru/l10n-pe-type-of-existence.png
+      :alt: Select Type of Existence for a product.
+
+- **Automatic inventory valuation:** For storable goods (products for which inventory is tracked),
+  it's important to use :doc:`automatic inventory valuation
+  <../../inventory_and_mrp/inventory/product_management/inventory_valuation/inventory_valuation_config>`.
+  Once automatic inventory valuation is turned on, this valuation method can be turned on for
+  a product's :ref:`Product Category <inventory/warehouses_storage/valuation-on-product-category>`.
+
+- **Costing method:** Storable goods also need a :doc:`costing method
+  <../../inventory_and_mrp/inventory/product_management/inventory_valuation/inventory_valuation_config>`
+  that is *not* set to `Standard Price`. This is important as the Journal Entries generated from
+  stock moves will feed the PLE reports.
```

Comment:
Update formatting, optional to use italics or bold here, using bold since it stands out more
```suggestion
  that is **not** set to `Standard Price`. This is important as the journal entries generated from
  stock moves feed the |PLE| reports.
```